content/references/rippled-api/transaction-formats/transaction-metadata.md

```diff
@@ -6,25 +6,67 @@ Transaction metadata is a section of data that gets added to a transaction after
 
 Some fields that may appear in transaction metadata include:
 
-| Field                                 | Value               | Description    |
-|:--------------------------------------|:--------------------|:---------------|
-| `AffectedNodes`                       | Array               | List of [ledger objects](ledger-object-types.html) that were created, deleted, or modified by this transaction, and specific changes to each. |
-| `DeliveredAmount`                     | [Currency Amount][] | **DEPRECATED.** Replaced by `delivered_amount`. Omitted if not a partial payment. |
-| `TransactionIndex`                    | Unsigned Integer    | The transaction's position within the ledger that included it. This is zero-indexed. (For example, the value `2` means it was the 3nd transaction in that ledger.) |
-| `TransactionResult`                   | String              | A [result code](transaction-results.html) indicating whether the transaction succeeded or how it failed. |
-| [`delivered_amount`](#delivered-amount) | [Currency Amount][] | The [Currency Amount][] actually received by the `Destination` account. Use this field to determine how much was delivered, regardless of whether the transaction is a [partial payment](partial-payments.html). [New in: rippled 0.27.0][] |
+{% include '_snippets/tx-metadata-field-table.md' %} <!--_ -->
```

Comment:
Comments on this file should go to #554, not here. (This file will leave the diff after I merge that PR and rebase.)